dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java

```diff
@@ -90,6 +90,18 @@ public static String getTimeMachinePath(){
 		return path;
 	}
 
+    public static String getStaticPublishPath(){
+
+        String path = Config.getStringProperty("STATIC_PUBLISHING_ROOT_PATH",
+                APILocator.getFileAssetAPI().getRealAssetsRootPath() + File.separator + "static_publishing");
+
+        File pathDir = new File(path);
```

Comment:
I think you might want to encapsulate this into the FileUtil.mkDirsIfNeeded (path)

Reply:
@jdotcms I can't find that method.